lakefile.lean

```diff
@@ -22,9 +22,58 @@ require YatimaStdLib from git
 require Cli from git
   "https://github.com/mhuisi/lean4-cli" @ "e70141d69b8562a0cd31d23a9c9a4f0f90a3c0a6"
 
+section Testing
+
 lean_lib Fixtures {
   roots := #[`Fixtures.AnonCidGroups.ToBeImported]
 }
 
 lean_exe Tests.AnonCidGroups { supportInterpreter := true }
 lean_exe Tests.Termination   { supportInterpreter := true }
+
+end Testing
+
+section Setup
+
+inductive CmdResult
+  | ok  : String → CmdResult
+  | err : String → CmdResult
+
+def runCmd (cmd : String) : ScriptM CmdResult := do
+  let cmd := cmd.splitOn " "
+  if h : cmd ≠ [] then
+    let (cmd, args) := match h' : cmd with
+      | cmd :: args => (cmd, ⟨args⟩)
+      | []          => absurd h' h
+    let out ← IO.Process.output {
+      cmd  := cmd
+      args := args
+    }
+    return if out.exitCode != 0
+      then .err out.stderr
+      else .ok out.stdout
+  else return .ok ""
+
+def getCurrDir : ScriptM String := do
+  match ← runCmd "pwd" with
+  | .ok res => return res.trim
+  | .err e  => panic! e
+
+def getHomeDir : ScriptM String :=
+  return s!"{"/".intercalate $ (← getCurrDir).splitOn "/" |>.take 3}"
+
+script setup do
+  IO.println "building yatima"
+  match ← runCmd "lake build" with
+  | .ok  _   =>
+    let mut binDir : String := s!"{← getHomeDir}/.local/bin"
+    IO.print s!"target directory for the yatima binary? (default={binDir}) "
```

Comment:
`IO.println` here?

Reply:
In this case it's `print` because it's a prompt and the user is supposed to answer on the same line